tests/integration/helpers.py

```diff
@@ -172,6 +173,7 @@ async def run_in_unit(unit: Unit, command: str, timeout=None) -> tuple[int, str
     Args:
         unit: Juju unit to execute the command in.
         command: Command to execute.
+        timeout: Amount of time to wait for the execution.
```

Comment:
How come this passed linting ealier?

Reply:
`pydocstyle` is configured to only go over `src`.

Reply:
Makes sense, I suppose